components/brave_wallet/common/fil_address.cc

```diff
@@ -190,6 +254,11 @@ std::string FilAddress::EncodeAsString() const {
   }
   std::vector<uint8_t> payload_hash(bytes_);
   std::vector<uint8_t> checksum(bytes_);
+  if (protocol_ == mojom::FilecoinAddressProtocol::DELEGATED) {
+    DCHECK(agent_id_.has_value());
+    auto agent_id = Leb128Encode(agent_id_.value());
+    checksum.insert(checksum.begin(), agent_id.begin(), agent_id.end());
+  }
```

Comment:
Can we assume and support only agent_id == 10? Or this is supposed to be extended in nearest future?
If agent_id == 10 always true then do we even need Leb128Encode?

Reply:
Ok, hardcoded 10